storage/byodb/snowflake-secure-data-sharing/index.md

```diff
@@ -0,0 +1,108 @@
+GRANT SELECT ON TABLE <DATABASE_NAME>.<SCHEMA_NAME>.<TABLE_NAME> TO SHARE <SHARE_NAME>;
+```
+
+To review object grants, use:
+```sql
+SHOW GRANTS TO SHARE <SHARE_NAME>;
+```
+
+### Add Accounts to the Share
+Add one or more accounts to the share using `ALTER SHARE`. To verify, use `SHOW SHARES`.
+```sql
+ALTER SHARE <SHARE_NAME> ADD ACCOUNTS=<ACCOUNT_NAME>;
+```
+```sql
+SHOW SHARES;
+```
+
+The share is now ready for consumption by the specified accounts. For detailed guidance, refer to [Create and configure shares](https://docs.snowflake.com/en/user-guide/data-sharing-provider).
+
+## Consumer Workflow
+To use the shared data, follow these steps to create and configure a database from the provided share. Refer to the [Snowflake documentation](https://docs.snowflake.com/en/user-guide/data-share-consumers) for more information.
+
+### Role Requirements
+To perform these tasks, use the `ACCOUNTADMIN` role or a role with the `IMPORT SHARE` global privilege. For more details, see [Enable non-ACCOUNTADMIN roles to perform data sharing tasks](https://docs.snowflake.com/en/user-guide/security-access-privileges-shares).
+
+### Limitations of Imported Databases
+Imported databases have the following restrictions:
+* **Read-only**: Consumers can query data but cannot modify it or create new objects.
+* Unsupported actions include:
+  * Cloning imported databases, schemas, or tables.
+  * Using Time Travel for imported databases or their objects.
+  * Editing comments for imported databases.
+* They cannot be:
+  * Re-shared with other accounts.
+  * Replicated.
+
+### Viewing Available Shares
+Use the web interface or SQL to view available shares:
+```sql
+SHOW SHARES;
+```
+
+```sql
+DESC SHARE <PRODUCER_ACCOUNT>.<SHARE_NAME>;
+```
+
+### Create a Database from a Share
+To consume shared data, create a database from the share:
+```sql
+CREATE DATABASE <CONSUMER_DATABASE_NAME> FROM SHARE <PRODUCER_ACCOUNT>.<SHARE_NAME>;
+```
+
+### Granting Privileges on an Imported Database
+Keboola currently supports only shares with [**direct access to database objects**](https://docs.snowflake.com/en/user-guide/data-sharing-policy-protected-data). Shares with roles are not supported.
+
+To allow users to access shared objects, grant the `IMPORTED PRIVILEGES` privilege on the imported database to one or more roles in your account. A role can grant `IMPORTED PRIVILEGES` only if it:
+* Owns the imported database (i.e., has the `OWNERSHIP` privilege).
+* Has been granted the `MANAGE GRANTS` global privilege.
+
+Example:
+```sql
+GRANT IMPORTED PRIVILEGES ON DATABASE <CONSUMER_DATABASE_NAME> TO <KEBOOLA_PROJECT_ROLE>;
+```
+
+## Final Step
+You can now register the schema in the newly created database as an external dataset in Keboola, enabling seamless data integration. Use `<CONSUMER_DATABASE_NAME>` as the database name and 
+the schema present in this database. When registering, don't forget to check the `Secure Data Share` checkbox.
```

Comment:
"Is Secure Data Share?" zni zvlastne. Navrhuju pouzit jenom "Secure Data Share", jestli souhlasis :-). Tooltip u toho vysvetluje, za jakych okolnosti to maji lidi zaskrtnout. Slo by to? Jestli jo, mohl bys to prosim upravit i v UI? 
Dekuju!

Reply:
Já to v UI neupravím. cc @kbl-tomas-hegedus

Reply:
Souhlas, vypadá to líp. Zadám ticket na UI.

Reply:
Takže tomu ofiko říkáme odteď SDS - Secure Data Share?

viz https://github.com/keboola/connection-docs/pull/698#discussion_r1846477797